src/identity.js

```diff
@@ -1677,6 +1677,10 @@ export default function Identity(mpInstance) {
         }
         mpInstance._Store.isInitialized = true;
 
+        if (mpInstance._RoktManager.isReady()) {
```

Comment:
So if it's not ready yet because identity returns faster than rokt manager gets initialized, this means there still won't be a current user right/. Is there a way to queue setting the current user for when it is ready?

Reply:
Actually is more of a check to see if the Rokt Kit is being used at all. We could potentially create a new method called `isConfigured` or `isActive`, but I felt like that might be unnecessary.

Reply:
ah got it.  yea seems like it's more readable/grokable if you go with an approach like isActive. I think you can make it a tech debt ticket.